app/views/notifications/_notification.haml

```diff
@@ -1,9 +1,12 @@
 .media.stream_element{:data=>{:guid => note.id, :type => (Notification.types.key(note.type) || '') }, :class => (note.unread ? 'unread' : 'read')}
   .unread-toggle.pull-right
     %i.entypo-eye{title: (note.unread ? t("notifications.index.mark_read") : t("notifications.index.mark_unread"))}
-  - if note.type == "Notifications::StartedSharing" && contact = current_user.contact_for(note.target)
-    .pull-right
-      = aspect_membership_dropdown(contact, note.target, "right")
+  - if note.type == "Notifications::StartedSharing" && (!defined?(no_aspect_dropdown) || !no_aspect_dropdown)
+    - contact = current_user.contact_for(note.target)
+    - if contact
+      - gon_load_contact(contact)
```

Comment:
`Notification` has `recipient` attriubute. Perhaps then implement `contact` method for `Notification::StartedSharing`:
```ruby
def contact
  recipient.contact_for(target)
end
```

?